docker/docker-compose.yml

```diff
@@ -183,6 +183,11 @@ services:
       context: ..
       dockerfile: docker/mock-pay/Dockerfile
     container_name: mock-pay
+    environment:
+      - IMPOSTER_LOG_LEVEL=TRACE
+
+    ports:
+      - "10001:8080"
```

Comment:
do we need to keep the ports?

Reply:
No, this is from experimenting with the mock endpoints, I'll drop 👍